jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js

```diff
@@ -70,7 +71,14 @@ YUI.add('deployment-signup', function() {
     */
     _handleSignup: function() {
       this._setBetaCookie(true);
-      console.error('Not implemented.');
+      this.props.changeState({
```

Comment:
Do we want to immediately navigate them when they click to open the new window? I'm just wondering if it might be confusing because to them they clicked to join the beta...are they now in the beta?